plugins/woocommerce/src/Internal/Admin/Orders/MetaBoxes/OrderAttribution.php

```diff
@@ -22,6 +22,36 @@ public function __construct() {
 		$this->set_fields_and_prefix();
 	}
 
+	/**
+	 * Format the meta data for display.
+	 *
+	 * @since 8.5.0
+	 *
+	 * @return void
```

Comment:
```suggestion
	 * @since 8.5.0
	 *
	 * @param array $meta The meta data.
	 * 
	 * @return void
```
This fails the linter as is.

❔  Also just curious why this method is pass by reference.

Reply:
In my mind it reads better. This is a method that modifies what you pass to it.

The alternative is:

```PHP
$meta = $this->format_meta_data( $meta );
```

Which is also fine. If you don't like the reference then I can change it 🤷🏻

Reply:
I have no preference, I was just curious whether you hold a strong opinion. Both strategies seem fairly common in the codebase, so we can leave it as is!